src/styles/global.scss

```diff
@@ -0,0 +1,60 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Global Styles
+ *
+ * ----------------------------------------------------------------------------- */
+
+:root {
+  //palette
+  --magenta: #cc4faa;
+  --tomato: #ed5338;
+  --aqua: #a3dad6;
+  --ocean: #0088ce;
+  --sunflower: #ffce34;
+  --geo: #00b18f;
+  --black: #000000;
+  --white: #ffffff;
+  --cloud: #e4e1dc;
+
+  //transitions
+  --easeOut: all 0.5s ease-out;
+
+  //fonts
+  --visueltBold: "Visuelt Bold", sans-serif;
+  --visueltRegular: "Visuelt Regular", sans-serif;
+  --scopeOne: "Scope One", sans-serif;
+  --ibmPlexMono: "IBM Plex Mono", monospace;
+
+  //font sizes
+  --headerFontSizePrimary: 64px;
+  --headerFontSizeSecondary: 36px;
+  --headerFontSizeTertiary: 22px;
+  --mobileTitleFontSize: 26px;
```

Comment:
Instead of defining separate variables for mobile/desktop, you can actually just reassign to the same variable via a media query. For instance:

```
:root {
  --titleFontSize: 64px;

  @media screen and (min-width: 768px) {
    --titleFontsize: 26px;
  }
}
```